src/components/SearchInput.js

```diff
@@ -32,7 +33,7 @@ class SearchInput extends React.Component {
     const styles = this.styles();
 
     return (
-      <div style={Object.assign({}, styles.component, this.props.style)}>
+      <div ref={this.props.searchInputRef} style={Object.assign({}, styles.component, this.props.style)}>
```

Comment:
@markstewy can you move this to the `Input` on the next line. You'll need to use `elementRef` (if that's what we've decided on).